packages/ocaml-monadic/ocaml-monadic.0.3.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "JHU PL Lab <pl.cs@jhu.edu>"
+authors: "JHU PL Lab <pl.cs@jhu.edu>"
+homepage: "https://github.com/zepalmer/ocaml-monadic"
+bug-reports: "https://github.com/zepalmer/ocaml-monadic/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/zepalmer/ocaml-monadic.git"
+build: [
+  ["oasis" "setup"]
```

Comment:
I'm still a bit of a novice at this, so please forgive my questions.  :)  If I understand correctly, you're suggesting that `setup.ml` be included in the tarball so that the build and build-test in the repo just rely on `setup.ml` itself?

We added `setup.ml` to our `.gitignore` because we didn't want to put generated content into our repository.  Is that not best practice in this scenario?  Thanks for your help; I'm eager to learn.  :)